INSTALL.md

```diff
@@ -178,16 +178,33 @@ make install
 
 In addition to meeting the [requirements](#requirements), you need a version of Visual Studio (the [community edition](https://www.visualstudio.com/vs/visual-studio-express/) is enough) and a CMake version that's compatible with the Visual Studio version you're using.
 
-You must also ensure that the msvc versions of dependencies you're using are placed in the correct folders.
+With the dependencies you have two options:
 
-For libsodium that is `c-toxcore/third_party/libsodium`, and for pthreads-w32, it's `c-toxcore/third_party/pthreads-win32`
+  - bootstrap vcpkg as described here https://github.com/microsoft/vcpkg#quick-start-windows and then do 
```

Comment:
The link should have this format(you can replace the text in [] with words):
``[https://github.com/microsoft/vcpkg#quick-start-windows](https://github.com/microsoft/vcpkg#quick-start-windows)```

Reply:
Done